cinema/views.py

```diff
@@ -0,0 +1,48 @@
+from django.shortcuts import get_object_or_404
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from rest_framework import status
+
+from cinema.models import Movie
+from cinema.serializers import MovieSerializer
+
+
+@api_view(["GET", "POST"])
+def movie_list(request):
+    """
+    List all movies, or create a new movie.
```

Comment:
comment seems unnecessary, it's clear that the function is named movie_list and the methods are GET and POST, so I can only get or create movies.

Reply:
Remove